src/utils/api.js

```diff
@@ -20,6 +22,10 @@ export const getMostRecentArticles = () => {
     .then((res) => res.data);
 };
 
+export const getAllTopics = () => {
+  return api.get(`/topics`).then((res) => res.data);
+};
+
 export const getArticlesByTopic = (topic) => {
```

Comment:
You have refactored 'getAllArticles' to accept a topic query, so you probably won't need this function here any more.